src/model/UMLClass.java

```diff
@@ -89,7 +91,7 @@ public void setName(String name) {
 	 * @return - true if class has method
 	 */
 	public boolean hasMethod(String methodName) {

```

Comment:
This method should take in the parameters as well as the key is built off of the parameters.